content/nginx-one/secure-your-fleet/secure.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Set up security alerts"
+weight: 500
+toc: true
+type: how-to
+product: NGINX One
+docs: DOCS-000
```

Comment:
```suggestion
nd-content-type: how-to
nd-product: NGINX One
```

Jon has asked us not to add placeholders for the docs parameter, since the absence is meaningful for the purposes of the automated script setting stuff up.